src/scraper/__main__.py

```diff
@@ -182,12 +245,133 @@ def scrape(self):
             if option.text
         }
         # if juidicial_officers param is not specified, use all of them
-        if not args.judicial_officers:
-            args.judicial_officers = list(judicial_officer_to_ID.keys())
+        if not judicial_officers:
+            judicial_officers = list(judicial_officer_to_ID.keys())
```

Comment:
❓ This is my python inexperience, but does `.keys()` already return a list?

Reply:
It's a view object for a dictionary's keys, but has to be set to a field if you want to call it later. It doesn't change the original field.

Reply:
I see! Boy my brain does not think in classes by default 😆